cognite/client/data_classes/sequences.py

```diff
@@ -697,6 +683,12 @@ def to_pandas(
         raise ValueError(f"Invalid key value '{key}', should be one of ['id', 'external_id']")
 
 
+class SequencesDataList(SequenceRowsList):
+    def __init__(self, resources: Collection[Any], cognite_client: CogniteClient | None = None) -> None:
+        super().__init__(resources, cognite_client)
+        warnings.warn("SequencesDataList is deprecated, use SequenceRowsList instead", DeprecationWarning)
+
+
```

Comment:
Ref. latest slack discussion, maybe just make this an alias?

Reply:
100% just hadn't updated it yet.